tests/TestPsci/TestEnv.hs

```diff
@@ -56,26 +56,40 @@ jsEval = liftIO $ do
     Just (ExitFailure _, _, err) -> putStrLn err >> exitFailure
     Nothing                      -> putStrLn "Couldn't find node.js" >> exitFailure
 
--- | Run a PSCi command and evaluate the output with 'eval'.
-runAndEval :: String -> TestPSCi () -> TestPSCi ()
-runAndEval comm eval =
+-- | Run a PSCi command and evaluate the output with 'eval', and the
+-- printed-to-console output with 'evalPrinted'
```

Comment:
This is still a bit unclear I think. What is the difference between the output and the printed-to-console output?